packages/office-ui-fabric-react/src/components/CommandBar/CommandBarPage.tsx

```diff
@@ -16,7 +16,7 @@ const CommandBarCustomizationExampleCode = require('!raw-loader!office-ui-fabric
 
 export class CommandBarPage extends React.Component<IComponentDemoPageProps, {}> {
   public render() {
-    let cmdBarParamsTextAndIcons: any = { items: items, farItems: farItems };
+    let cmdBarParamsTextAndIcons: any = { items: items, overflowItems: overflowItems, farItems: farItems };
```

Comment:
Hey one minor suggestion, in the latest typescript, you can shorten this using a shorthand syntax:

```tsx
{ items, overflowItems, farItems }
```

Which is equivalent to what's there. "item: item" can be collapsed to just "item"